### DISCUSSION TITLE
Option to correct the assistant response

### DISCUSSION THREAD

**Discussion by XavierRex** [LAION-AI/Open-Assistant#3460] [Ideas]
It would be useful to have an option to be able to correct an assistant response for some time. Often an error is noticed after it is already added. I think 15 min will be enough.

**Comment by ighoshsubho** [comment]
Hey can you please assign me the issue, I'm thinking of a simple approach of keeping a record of the last few responses given by the assistant using a queue and would timeout every 15 mins.

**Comment by andrewm4894** [comment]
is this related to this? 

https://github.com/LAION-AI/Open-Assistant/issues/2832

if not i'm a little unsure what this issue is about - eg if is some technical issue or related to ML side of things.

**Comment by IllusionDX** [comment]
It would be great if they could add a better feedback system for the chat's responses. Sometimes it makes incorrect answers and the upvote/downvote just doesn't cut it. An option to flag it and send a correction like in ChatGPT would be great. It could even be even be made into a task on the dashboard
